ddtrace/contrib/django/__init__.py

```diff
@@ -58,6 +58,10 @@
   be useful if you want to use the Django integration, but you want to trace only
   particular functions or views. If set to False, the request middleware will be
   disabled even if present.
+* ``DATABASE_AUTO_INSTRUMENT`` (default: ``True``): if set to false database will not
```

Comment:
I think is better having `AUTO_INSTRUMENT_DATABASE` so it's compliant with settings of other integrations / languages. Pointing out that `AUTO_INSTRUMENT` must be true is great! thank you!